content/3_Silver/Flood_Fill.mdx

```diff
@@ -739,3 +739,23 @@ print(room_num)
 ## Problems
 
 <Problems problems="general" hideSuggestProblemButton />
+
+## Quiz
+
+<Quiz>
+	<Quiz.Question>
+		What would be a valid function header for DFS on an undirected graph to work *without* any other variables?
+		<Quiz.Answer correct>
```

Comment:
Wouldn't both the first and second answer work? The second one is just much slower

Reply:
doesn't it get stuck in a dead loop unless you have an external visited array?

Reply:
nevermind you're right

Reply:
ok so i specifically said O(N) now